models/actions/artifact.go

```diff
@@ -149,3 +155,16 @@ func ListArtifactsByRunIDAndName(ctx context.Context, runID int64, name string)
 	arts := make([]*ActionArtifact, 0, 10)
 	return arts, db.GetEngine(ctx).Where("run_id=? AND artifact_name=?", runID, name).Find(&arts)
 }
+
+// ListExpiredArtifacts returns all expired artifacts but not deleted
+func ListExpiredArtifacts(ctx context.Context) ([]*ActionArtifact, error) {
+	arts := make([]*ActionArtifact, 0, 10)
+	return arts, db.GetEngine(ctx).
+		Where("expired_unix < ? AND status = ?", timeutil.TimeStamp(time.Now().Unix()), ArtifactStatusUploadConfirmed).Find(&arts)
```

Comment:
`ArtifactStatusUploadConfirmed` but not `ArtifactStatusUploadExpired`?

Reply:
I need change the name `ListExpiredArtifacts` to `ListNeedExpiredArtifacts`. It gets over expire time but not expired status records.